pkg/parser/antlr/FqlParser.g4

```diff
@@ -223,8 +223,37 @@ namespace
     : (NamespaceSegment)*
     ;
 
+functionIdentifier
```

Comment:
`Use`, `Or`, `And` is not a function identifiers. Maybe there is better name for it? `Keyword`?

Reply:
``TEST::USE()`` is a function :)

Reply:
I understand :) I missed `: Identifier` at the beginning and thought there was only a list of keywords.  Anyway, why `TRUE` and `FALSE` not on this list?)

Reply:
They are. ``BooleanLiteral``.